dir.props

```diff
@@ -202,6 +186,25 @@
     <DefineConstants>$(DefineConstants),TRACE</DefineConstants>
   </PropertyGroup>
 
+  <!-- Provide defaults for TestNugetRuntimeId -->
+  <PropertyGroup Condition="'$(TestNugetRuntimeId)'== ''">
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='Windows_NT'">win7-x64</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='Linux'">ubuntu.14.04-x64</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='OSX'">osx.10.10-x64</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='FreeBSD'">ubuntu.14.04-x64</TestNugetRuntimeId>
```

Comment:
The honest answer is I don't know.  My guess is in the limit the managed code is going to be compatible (since it will just call into the native shims) and the native code can be overwritten with the `BUILDTOOLS_RUNTIME_OVERRIDE` stuff.

It is certainly better to pass this an windows RID, but I suspect that using the OSX RID would be fine as well.  Once FreeBSD is further along and we are producing actual FreeBSD packages, we will want to change this, but for now it is reasonable.